Recursive/FactorialCalculator.js

```diff
@@ -1,5 +1,5 @@
-const factorialRecursion = (n) => n === 0 ? 1 : n * factorialRecursion(n - 1);
+const factorialRecursion = (n) => n === 0 ? 1 : n * factorialRecursion(n - 1)

```

Comment:
Don't use arrow function, write it in the form of a normal function instead

Reply:
Not sure what's the problem with arrow functions, but sure.